CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 #### Features
 
 * Your contribution here.
+* [#1943](https://github.com/ruby-grape/grape/pull/1943): Regex string allocation - [@ericproulx](https://github.com/ericproulx).
```

Comment:
Nitpick, this should describe what it does, eg. "Reduce number of regex string allocations"